lib/ansible/modules/system/lvg.py

```diff
@@ -53,8 +53,8 @@
     description:
     - Control if the volume group exists.
     type: str
-    choices: [ absent, present ]
-    default: present
+    choices: [ "absent", "present" ]
+    default: "present"
```

Comment:
quotes are not necessary for string, could you revert these?

Reply:
while YAML supposedly can work with and without quotes this introduces ambiguity where I have seen some YAML parser failing (outside of Ansible world tho), so I had to employ as a good practice always put quotes around strings in any YAML documents.

If this is a requirement to NOT have quotes around string fields, then sure I can revert this.

Reply:
it is not about yaml here, I totally unerstand. We accept quoted or unquoted strings but generally prefer quotes only where needed. 

However, achange that is not required or does not "fix" anything. Good practice (opinions differs from person to person what good practice is) is not a reason to change a thing that have worked. Thanks for your understanding

Reply:
done